src/components/molecules/Article/index.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import Avatar from "@atoms/Avatar";
+import { IoMdEye as EyeIcon } from "react-icons/io";
+import { MdModeComment as CommentIcon } from "react-icons/md";
+import CommentContainer from "@molecules/CommentContainer";
+import { displayDate } from "~/utils";
+import { User, Article as IArticle, Comment } from "~/types";
+import "./style.scss";
+
+export interface ArticleTitleProps {
+    article: Omit<IArticle, "comment" | "contents">;
+}
+
+export const ArticleTitle = (props: ArticleTitleProps) => {
+    const {
+        author,
+        createdAt,
+        updatedAt,
+        title,
+        commentCount,
+        viewCount,
+    } = props.article;
+    const dateView = displayDate(updatedAt || createdAt, {
+        format: "YYYY/MM./DD A hh:mm",
+    });
+
+    const { avatar = "", github = "", username = "" } = author;
+
+    return (
+        <div className="box_post_title">
+            <div className="title_info">
+                <h1>{title}</h1>
+                <div>
+                    <time>{dateView}</time>
+                    <strong>{username}</strong>
```

Comment:
![image](https://user-images.githubusercontent.com/22635168/78800754-c5764680-79f6-11ea-8736-e541d8ea11f7.png)

[strong tag](https://developer.mozilla.org/ko/docs/Web/HTML/Element/strong)

꼭 중요도를 높여야 하는 내용이 아니라면 b 태그를 이용해보는건 어떤가요?
```suggestion
                    <b>{username}</b>
```